scripts/rollup/forks.js

```diff
@@ -345,7 +389,23 @@ const forks = Object.freeze({
         if (!rendererInfo.isServerSupported) {
           return null;
         }
-        return `./packages/react-server/src/forks/ReactFlightServerConfig.${rendererInfo.shortName}.js`;
+        if (rendererInfo.isFlightSupported === false) {
+          return new Error(
+            `Expected not to use ReactFlightServerConfig with "${entry}" entry point ` +
+              'in ./scripts/shared/inlinedHostConfigs.js. Update the renderer config to ' +
+              'activate flight suppport and add a matching fork implementation for ReactFlightServerConfig.'
+          );
```

Comment:
previously this would return a fork to a non-existent file such as `ReactFlightServerConfig.dom-fb.js` but since that file was never referenced there was no error. To mimic this we return an error here when we don't fine a file as long as the bundle declares that it does not implement Flight.